actionsList.ts

```diff
@@ -0,0 +1,68 @@
+interface IToDo {
+    name: string,
+    date: Date,
+    finished: boolean,
+}
+
+class Action implements IToDo {
+    constructor(public name: string, public date: Date, public finished: boolean) {
+    }
+
+    public finishTask() {
+        this.finish();
+    }
+
+    private finish() {
+        this.finished = true;
+    }
+}
+
+class ActionsList {
+    public toDos = [];
+
+    public addToDo(Action: object) {
```

Comment:
Parametrul Action trebuie scris cu litera mica, adica vorbim despre un parametru x de tip y, adica x: y, in cazul tau: `action: Action`